### ISSUE TITLE
Update Estonian translations

### ISSUE THREAD

**Issue by Eilyre** [squidfunk/mkdocs-material#7262]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "et",
  "action.edit": "Muuda seda lehte",
  "action.skip": "Keri sisuni",
  "action.view": "Vaata lehe lähtekoodi",
  "announce.dismiss": "Ära näita uuesti",
  "blog.archive": "Arhiiv",
  "blog.categories": "Kategooriad",
  "blog.categories.in": "sees",
  "blog.continue": "Jätka lugemist",
  "blog.draft": "Mustand",
  "blog.index": "Tagasi indeksisse",
  "blog.meta": "Metaandmed",
  "blog.references": "Seotud lingid",
  "clipboard.copy": "Kopeeri lõikelauale",
  "clipboard.copied": "Kopeeritud",
  "consent.accept": "Nõustu",
  "consent.manage": "Halda seadeid",
  "consent.reject": "Keeldu",
  "footer": "Jalus",
  "footer.next": "Järgmine",
  "footer.previous": "Eelmine",
  "header": "Päis",
  "meta.comments": "Kommentaarid",
  "meta.source": "Lähtekood",
  "nav": "Navigatsioon",
  "readtime.one": "1-minutiline lugemine",
  "readtime.other": "#-minutiline lugemine",
  "rss.created": "RSS-voog",
  "rss.updated": "Uuendatud sisu RSS-voog",
  "search": "Otsing",
  "search.placeholder": "Otsi",
  "search.share": "Jaga",
  "search.reset": "Lähtesta",
  "search.result.initializer": "Otsingu lähtestamine",
  "search.result.placeholder": "Otsimiseks kirjuta siia",
  "search.result.none": "Otsingule ei leitud ühtegi vastet",
  "search.result.one": "Leiti üks tulemus",
  "search.result.other": "Leiti # tulemust",
  "search.result.more.one": "1 veel sellel lehel",
  "search.result.more.other": "# veel sellel lehel",
  "search.result.term.missing": "Puudub",
  "select.language": "Vali keel",
  "select.version": "Vali versioon",
  "source": "Ava repositooriumis",
  "source.file.contributors": "Autorid",
  "source.file.date.created": "Loodud",
  "source.file.date.updated": "Viimane uuendus",
  "tabs": "Vaheleht",
  "toc": "Sisukord",
  "top": "Üles"
}[key] }}{% endmacro %}
```

### Country flag

:flag_ee:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Awesome, thanks!

**Comment by squidfunk** [comment]
Updated translations in f4db9de0f018427ceb6bc28d5fb095d6c1468b13, tagged you as a co-author 🚀

**Comment by squidfunk** [comment]
Released as part of [9.5.27](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.27).
